src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,480 @@
+          :key="'available-option-' + id + '-' + option.id"
+          :id="'available-option-' + id + '-' + option.id"
+          :class="optionsClass(option, 'available')"
+          @click="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.space.prevent.stop="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'available', 'chosen')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'available', 'chosen')"
+          @keyup.right.prevent.stop="onMoveSelectedClick('available')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick('available')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+        {{ option.displayText }}
+        </li>
+      </ul>
+    </div>
+
+    <div class='buttons'>
+      <button 
+        @click="onMoveAllOptions('available')"
+        id='all-to-chosen'
+        type='button'
+        >
+        <img class="icon" :src="allToChosen" />
+      </button>
+      <button 
+        @click="onMoveSelectedClick('available')"
+        id='selected-to-chosen'
+        type='button'
+        >
+        <i class='icon ion-arrow-right-c' aria-hidden='true'></i>
+      </button>
+      <button
+        @click="onMoveSelectedClick('chosen')"
+        id='selected-to-available'
+        type='button'>
+        <i class='icon ion-arrow-left-c' aria-hidden='true'></i>
+      </button>
+      <button 
+        @click="onMoveAllOptions('chosen')"
+        id='all-to-available'
+        type='button'
+        >
+        <img class="icon" :src="allToAvailable" />
+      </button>
+    </div>
+
+    <div class='chosen'>
+      <label id='chosen-label'>{{ chosenLabel }}</label>
+      <ul
+        aria-labeledby='chosen-label'
+        role='selectbox'
+      >
+        <li
+          v-for="option in chosenOptions"
+          :key="'chosen-option-' + id + '-' + option.id"
+          :id="'chosen-option-' + id + '-' + option.id"
+          :class="optionsClass(option, 'chosen')"
+          @click="onOptionClick(option, 'chosen', 'available', $event)"
+          @keyup.space.stop.prevent.stop="onOptionClick(option, 'chosen', 'available', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'chosen', 'available')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'chosen', 'available')"
+          @keyup.left.prevent.stop="onMoveSelectedClick('chosen')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick('chosen')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+          {{ option.displayText }} 
+
+        </li>
+      <span v-show="errors.has(validationId)" class="error-text" :id="'error-' + id">{{ errors.first(validationId) }}</span>
```

Comment:
Is this span supposed to be in the `ul`? And if so it should be tabbed in

Reply:
its the error message, it belongs there

Reply:
It belongs in the unordered list? `span` is technically not a valid child element of a `ul`

Reply:
i like the user experience with it

Reply:
So does this show within the shuttle instead of below it? It would be more consistent with other errors if it showed below.

Reply:
fine